### PR TITLE
chore: updating cached dropgz to v0.1.3

### PR DESCRIPTION
**Pull request by rjdenney**
**What type of PR is this?**
kind feature

**What this PR does / why we need it**:
chore: updating cached dropgz to v0.1.3
dropgz v0.1.3 has fixes for Windows Dualstack Overlay
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -67,11 +67,11 @@
       "multiArchVersions": [
         "v0.0.4.1",
         "v0.0.15",
-        "v0.1.1"
+        "v0.1.3"
       ],
       "prefetchOptimizations": [
         {
-          "version": "v0.0.15",
```
Comment (cameronmeissner) [inline vhdbuilder/packer/components.json:74]: we probably want to keep prefetching 0.0.15 if RP hasn't switched over to using 0.1.3 everywhere as of yet

Comment (rjdenney) [reply vhdbuilder/packer/components.json:74]: Once this merges in we want to change RP to 0.1.3. But if we need to have it in RP first I can make them separate PRs. 

Our current workstream for upgrading our images is AgentBaker before RP

Comment (cameronmeissner) [reply vhdbuilder/packer/components.json:74]: that's fine, let's keep prefetching 0.0.15 once the corresponding RP change has rolled out to all prod regions then

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -67,7 +67,7 @@
       "multiArchVersions": [
         "v0.0.4.1",
         "v0.0.15",
-        "v0.1.1"
+        "v0.1.3"
       ],
       "prefetchOptimizations": [
```
[the remark this answers is not in the document]
Comment (rjdenney) [reply vhdbuilder/packer/components.json:72]: Even though it isn't in RP yet?

[the remark this answers is not in the document]
Comment (rjdenney) [reply vhdbuilder/packer/components.json:72]: I added it back to prefetch

[the remark this answers is not in the document]
Comment (cameronmeissner) [reply vhdbuilder/packer/components.json:72]: yes that's fine, but we should be prefetching both

[the remark this answers is not in the document]
Comment (rjdenney) [reply vhdbuilder/packer/components.json:72]: just added it so it is both are being prefetched.

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**